src/docs/swagger.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "version": "1.0.0",
     "title": "Authors Heaven",
-    "description": "Authors Heaven Appi Documentation",
+    "description": "usersors Heaven Appi Documentation",
```

Comment:
@truestbyheart why is this still `usersors`???
Please, go over your PR changes carefully so I can merge.